app/lookups/land/user_agent_type.rb

```diff
@@ -2,7 +2,7 @@ module Land
   class UserAgentType < ApplicationRecord
     include TableName
 
-    lookup_by :user_agent_type, cache: true
+    lookup_by :user_agent_type, cache: 50, find_or_create: true
```

Comment:
What is the reasoning for this change?

Reply:
I added this change because without the `find_or_create` the `UserAgentType` was not being set in the `user_agent` method. When testing with a pry in the `user_agent` method `UserAgentType.all ` would return an empty array, so anytime I set the `UserAgentType` it would return `nil`!

Reply:
This ended up fixing it, because without the `find_or_create: true`, `UserAgentType['user']` would raise a `NoMethodError`. So adding this line enabled `UserAgentType['user]` to be found or created to then be set on the `UserAgent` instead of returning `nil`. 
 
 Updating the cache from true to 50 ended up fixing the problem as well because `cache: true` was returning an error when combined with `find_or_create: true`. Because `lookup_by` can only be called once per model I ended up receiving a `RuntimeError` when `cache: true` was set. This is because `lookup_for :user_agent_type` was already called in the `UserAgent` model causing this error. Using `cache: 50` allows `lookup_by` to use a separate LRU cache and is able to handle the `lookup_for` in the `UserAgent` model.